packages/components/icons/README.md

```diff
@@ -72,3 +72,36 @@ The `data` passed to the component is run through a DOM sanitizer to prevent unw
 ### Where to use
 
 This component can be used whenever the icon has to be rendered dynamically on runtime. For example in the Merchant Center this can be the case for the navigation menu icons, etc.
+
+## Leading Icon
+
+For use when you need to render an icon with a themed bounding-box. Icons can be either from the default set of icons, or custom SVG's.
```

Comment:
@FilPob Do you have any other tip we should include here about what's the purpose of this new component?

Reply:
Sure, I would say something like: The leading icon is a an eye-catchy visual element that should be used when an additional visual prominence is needed for a content section in the UI. The different colours in combination with the icons can be utilised to create certain categorisation of the elements in the UI.

Reply:
updated to Filip's description [here](https://github.com/commercetools/ui-kit/pull/2701/commits/ec6296c92de660bc69dbf14453079f7433827243)